Fix protocols faq (#3679)
segmentio/segment-docs: src/protocols/faq.md

### Can I duplicate a Tracking Plan in the Segment UI?

> You can duplicate Tracking Plans in the Segment web app. Here are the instructions to [copy a tracking plan](/docs/protocols/tracking-plan/create/#copy-a-tracking-plan). You can also use the [Tracking Plan API](/docs/protocols/apis-and-extensions/) to copy the underlying JSON schema from one Tracking Plan to another.

Reviewer: ```suggestion
Yes, you can duplicate Tracking Plans in the Segment web app by following the [instructions to copy a tracking plan](/docs/protocols/tracking-plan/create/#copy-a-tracking-plan). You can also use the [Tracking Plan API](/docs/protocols/apis-and-extensions/) to copy the underlying JSON schema from one Tracking Plan to another.
```

Reviewer: @ranand is the feature to duplicate tracking plans currently in beta or did it already go to GA. If it's in public beta, we'll need to call that out here. When's it going to GA?
